packages/manager/apps/cloud/client/app/app.js

```diff
@@ -130,7 +130,7 @@ angular
       const dedicatedRedirections = [
         '/paas/veeam-enterprise',
         '/paas/veeam',
-        '/iaas/vps',
+        '/vps',
```

Comment:
IMHO, `vps` url in `cloud` always followed `/iaas/vps*`, so we should keep this redirection pattern as this (if someone had a bookmark on it for example, but nobody have `/vps` bookmark on `cloud`). And we should redirect from `/iaas/vps` to `/vps` in `dedicated` for same reasons.


Redirection should works like this : `cloud/#/iaas/vps` => `dedicated/#/iaas/vps` => `dedicated/#/vps`

What do you think ?

Reply:
@cbourgois I agree, I did not pay attention while doing a batch replace but you're right, I'll update